components/Modal.jsx

```diff
@@ -0,0 +1,34 @@
+import { useRef, useEffect } from "react";
+import styles from "@/styles/components/modal.module.scss";
+
+export default function Modal({ content, isModalOpen, setIsModalOpen }) {
```

Comment:
Use `children` instead of `content`. There is more than semantical aspect to it - 
https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children

Reply:
noted